docs/source/events/payment-status-change.rst

```diff
@@ -6,6 +6,30 @@ This event means that the status of a payment has changed. To reduce noise, we o
 For each payment method, there are typical status sequences that the payment goes through. The exact meaning of the statuses and content of the ``payment_data`` also depends on the payment method.
 
 
+Line items
+----------
+
+.. versionadded:: 1.2.0
```

Comment:
I assume `versionadded` is in the correct case here but maybe not?

Reply:
Yep, that has a syntactic meaning in RST/Sphinx and gets expanded into an english sentence like: Added in version 1.2.0.